clients/imodels-client-management/README.md

```diff
@@ -4,59 +4,70 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See [LICENSE.md
 
 ## About this package
 
-This package contains an API client that exposes a subset of [iModels API](https://developer.bentley.com/apis/imodels/) operations that enable applications to manage iModels - query Changesets, Locks and other related entities, create Named Versions, etc. This is a lightweight library intended to be used by iTwin management applications that do not write any data into the iModel itself, an example of such application is the [iTwin Demo Portal](https://itwindemo.bentley.com/).
+This package contains an API client that exposes a subset of [iModels v2 API](https://developer.bentley.com/apis/imodels-v2/) operations that enable applications to manage iModels - query Changesets, Locks and other related entities, create Named Versions, etc. This is a lightweight library intended to be used by iTwin management applications that do not write any data into the iModel itself.
```

Comment:
> [iModels v2 API](https....

here I think the version specifier is not needed, the API is still called iModels API, this client just targets a particular version. "iModels v2 API" to me sounds like a new separate API independent of "iModels API"